site.yml

```diff
@@ -20,6 +20,7 @@
   environment: '{{ env }}'
   roles: 
   - { role: base }
+  - { role: dev, run_as: master }
```

Comment:
hey @erikh , sorry to be a stubborn about this but I don't think we should merge this in `ansible/master`. 

Let me explain, `dev` role is meant to be run only during build due to one fundamental assumption around it. Basically it is designed to include all **other** roles as it's dependencies and then it assumes that ansible is run with `prebake-for-dev` tag at command-line which filters and runs only the the tasks marked as `prebake-for-dev`.

By bringing it in volplugin we are going to end-up pulling/running a lot of unwanted tasks, which we don't want. Also this will be maintenance nightmare as now we are overloading the `dev` role to be run in more than one situation (with and without `prebake-for-dev` tag).

A simpler and cleaner approach to address what we hit in volplugin yesterday would be to instead publish a new box based on new ansible and use that instead.